avocado/utils/nvme.py

```diff
@@ -326,3 +327,63 @@ def get_free_space(controller_name):
         if line.split(":")[0].strip() == "unvmcap":
             return int(line.split(":")[-1].strip())
     return 0
+
+
+def create_namespaces(controller_name, ns_count):
+    """
+    creates eaqual n number of namespaces on the specified controller
+
+    :param controller_name: name of the controller like nvme0
+    :param ns_count: number of namespaces to be created
+    """
+    namespaces = get_current_ns_ids(controller_name)
+    if namespaces:
+        delete_all_ns(controller_name)
```

Comment:
I rethought about this after our last discussion,  I feel we should not by default delete while creating new..  so better to keep the delete util separate and also if user wants they can call delete before create call

Reply:
somehow i like the older approach as if namespaces are present  it should delete namespaces to avoid any issue ,

Reply:
i get that but still somehow i feel other way , maybe we can have a class readme

Reply:
@Naresh-ibm had discussion with naresh offline, we have agreed to delete ns in the util,  go with the old changes

Reply:
we are good here